### ISSUE TITLE
Create "Overlay" container

### ISSUE THREAD

**Issue by peasandwell** [HeathWallace/coinface#112]
This is related to #111 ``<TotalsCalendar />`` component, the ``<Overlay />`` container will allow its children to be rendered in an overlay. The child component should be positioned in the middle of the screen.

See the Totals Calendar for an example of usage - https://heathwallace.invisionapp.com/share/ENFXAUAMSQY#/screens/283473504

**Comment by jezzipin** [comment]
I would like to work on this.

**Comment by dan1elhughes** [comment]
Yup, additionally it'd be nice if the fade background could be clickable to call `onClose()` too
